src/components/NavGrid.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <section class="nav__grid">
+        <a v-for="item in items"  class="nav__box"
+        :href="item.link"
+        :style="'background: url(' + item.img + ');background-size: cover; display: block;'">
+            <div class="nav__description">{{ item.description }}</div>
+        </a>
+    </section>
+</template>
+
+
+<style>
```

Comment:
Try use `<style scoped>` where possible to restrict the styling to this particular component.